### PR TITLE
Docs(ai): Add Cost strategy to Ai Rate Limiting plugin 

### PR DESCRIPTION
**Pull request by AntoineJac**
### Description

Add Cost strategy to Ai Rate Limiting plugin 
 

### Testing instructions

Preview link: 

### Checklist 

- [ ] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@AntoineJac Hi! Thanks for making this PR! Is this change for 3.8?

**Comment by AntoineJac** [comment]
@cloudjumpercat  yes indeed. The feature has been pushed to this PR is ready to merge

File: app/_hub/kong-inc/ai-rate-limiting-advanced/overview/_index.md [hunk 1]
```diff
@@ -94,10 +94,13 @@ The plugin supports three strategies to calculate the token:
...
+| `total_tokens`   | Represents the total number of tokens, including both the prompt and the generated completion, in the LLM's input sequence. |
+| `prompt_tokens` | Represents the tokens provided by the user as input to the LLM, typically defining the context or task. |
+| `completion_tokens`   | Represents the tokens generated by the LLM in response to the prompt, representing the completed output or continuation of the task. |
 
+{% if_version gte: 3.8.x %}
+| `cost`   | Represents the financial or computational cost incurred based on the tokens used by the LLM during the request. Using this strategy can help you limit API usage based on the actual costs of processing the request, ensuring that expensive requests (in terms of token usage) are managed more carefully. This cost is calculated by taking the sum of multiplying the number of prompt tokens by the cost per prompt token (input cost) and by multiplying the number of completion tokens by the cost per completion token (output cost): `cost = prompt_tokens * input_cost + completion_tokens * output_cost`. The `input_cost` and `output_cost` fields must be defined in the AI plugins for this to work. |
```
Comment (cloudjumpercat) [inline app/_hub/kong-inc/ai-rate-limiting-advanced/overview/_index.md:102]: > The `input_cost` and `output_cost` fields must be defined in the AI plugins for this to work.

@AntoineJac Are those fields in this AI Rate Limiting Advanced plugin, or are these in a different AI plugin(s)?

Comment (AntoineJac) [reply app/_hub/kong-inc/ai-rate-limiting-advanced/overview/_index.md:102]: @cloudjumpercat  it is a new field pushed in 3.8 for the AI Rate Limiting Advanced plugin

Comment (cloudjumpercat) [inline app/_hub/kong-inc/ai-rate-limiting-advanced/overview/_index.md:102]:
```suggestion
| `cost`   | Represents the financial or computational cost incurred based on the tokens used by the LLM during the request. Using this strategy can help you limit API usage based on the actual costs of processing the request, ensuring that expensive requests (in terms of token usage) are managed more carefully. This cost is calculated by taking the sum of multiplying the number of prompt tokens by the cost per prompt token (input cost) and by multiplying the number of completion tokens by the cost per completion token (output cost): `cost = prompt_tokens * input_cost + completion_tokens * output_cost`. The `input_cost` and `output_cost` fields must be defined in the AI Rate Limiting Advanced plugin for this to work. |
```

**Review by cloudjumpercat — APPROVED** [review]
LGTM! Thanks for adding this!

**Pull request merged**